src/runtime/runtime.js

```diff
@@ -816,6 +824,23 @@ export class ConfiguredRuntime {
     this.buttonApi_.attach(button, optionsOrCallback, opt_callback);
   }
 
+  /** @override */
+  renderSmartButton(container, options, callback) {
+    if (!options) {
+      Object.assign({}, {theme: Theme.LIGHT, lang: 'en'});
```

Comment:
Hmm. I don't think this is doing anything. Plus the `if (!options)` is also wrong I believe. Per method's signature, the you should be expecting one one of these shapes: 

```
renderSmartButton(container, callback)
```
or
```
renderSmartButton(container, options, callback)
```

Thus, the second argument is not guaranteed to be an options map. And it's never not specified.

Reply:
Changed the supported signature for `renderSmartButton`